Functions.Templates/Templates/DurableFunctionsActivity-JavaScript/index.js

```diff
@@ -0,0 +1,7 @@
+﻿/*
+ * Before running this sample, please create a Durable Orchestrator function
```

Comment:
We mention "before running this sample" here and elsewhere, but really they won't ever run it directly. Would it be clearer to say "This function is not intended to be invoked directly. Instead it will be triggered by an orchestrator function.", or something along those lines?